### PR TITLE
DXL Image

### PR DESCRIPTION
**Pull request by esharf**

### CODE REVIEW

File: docker/dxl/Pipfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+dxlclient = "*"
+dxlmarclient = "*"
+dxltieclient = "*"
+virtualenv = "*"
```
Comment (glicht) [inline docker/dxl/Pipfile]: Are you sure you need this?

Comment (esharf) [reply docker/dxl/Pipfile]: I'm not sure about all the packages, but it's a copy from [demisto/dxl:latest](https://hub.docker.com/r/demisto/dxl/tags).
its an updated docker for dxl integration.
[#7571](https://github.com/demisto/etc/issues/7571)

Comment (glicht) [reply docker/dxl/Pipfile]: remove the virtualenv then
```suggestion
```

File: .dependabot/config.yml [hunk 2]
```diff
@@ -643,3 +643,9 @@ update_configs:
     - match:
         dependency_name: demisto/*
         update_type: all
+        
+  - package_manager: python
+    directory: /docker/dxl
+    update_schedule: live
+  - package_manager: docker
+    directory: /docker/dxl
```
Comment (glicht) [inline .dependabot/config.yml]: You are missing the settings for schedule and automerge. The script should have added this automatically. Please modify:
```suggestion
    directory: /docker/dxl
    update_schedule: daily
    automerged_updates:
    - match:
        dependency_name: demisto/*
        update_type: all
```

**Comment by glicht** [comment]
Looks good. You can merge

**Pull request merged**